src/libcollections/string.rs

```diff
@@ -317,9 +317,14 @@ impl String {
 
     /// Creates a new `String` from a length, capacity, and pointer.
     ///
-    /// This is unsafe because:
+    /// # Unsafety
     ///
-    /// * We call `Vec::from_raw_parts` to get a `Vec<u8>`;
+    /// This is _very_ unsafe because:
+    ///
+    /// * We call `Vec::from_raw_parts` to get a `Vec<u8>`. Therefore, this
+    ///   function inherits all of its unsafety, see [its
+    ///   documentation](../collections/string/struct.String.html#method.from_raw_parts)
```

Comment:
Ahh, possibly, yes. I just looked at the URLs and compared them... also this is to `String`, not `Vec`, which I looked at like five times, sigh